index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html> 
+<html lang="en">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>First Mobile Page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <main id="main">
+    <header class="first">
+        <div class="first-img"><img src="./images/Welcome.png" alt="welcome button at the top left"></div>
+        <nav><div class="first-img" id="hamburger" > <img src="./images/Union.png" alt="a union of three vectors"> </div></nav>
+    
+        <ul class="header-first">
+         <li>Portfolio</li>
+         <li>About </li> 
+         <li>Contact </li>
+        </ul>
+     
+
+    </header>
+    <br>
+    <section class="full-section hide" id="portfolio">
+        <div class="mobile-menu-section">
+            <div class="mobile-menu">
+                <div class="close-button" id="close" > <img src="./images/close.svg" alt="a union of three vectors"> </div>
+                <ul class="mobile-menu-ul">
+                    <li  class="mobile-menu-option"><a href="#portfolio">Portfolio</a></li>
+                    <li  class="mobile-menu-option"><a href="#about">About</a> </li> 
+                    <li  class="mobile-menu-option"><a href="#contact">Contact</a> </li>
+                   </ul>
+            </div>
+        </div>
+        <div class="dash"></div>
+</section>
+    <section class="second">
```

Comment:
- [ ] As stated 😃 here by the previous reviewer, the font size of h1 and p is not like the Figma 👌I see that you give the gave the h1 and p tags property size: of 40px and 20px but it is not working💯, so try to write it like font-size not size only to let it work 👍

| Figma design | Your design| 
| --- | --- |
|   ![image](https://user-images.githubusercontent.com/83360636/212319546-3c3c32a5-8ed9-423f-836b-da06651a0db2.png) | ![image](https://user-images.githubusercontent.com/83360636/212319315-2b2b909c-6f98-431b-b48a-3296f7141bd8.png) |